admin/about_me.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+require 'functions.php';
+
+$db = connectDatabase();
+$stmt = $db->query("SELECT `bio_title`,`bio`,`contact_title`,`email`,`telephone` FROM `about_me`;");
+$about_me_data = $stmt->fetch();
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>About me</title>
+</head>
+<body>
+
+    <form method="POST" action="submit_about_me.php">
+        Bio title: <input type="text" name="bio_title" value="<?php echo $about_me_data["bio_title"] ?>"><br/>
```

Comment:
Lines 20 to 24 consider using labels instead of free text

Reply:
changed to labels

Reply:
<p>'s

Reply:
p's